test/cases/ostree-simplified-installer.sh

```diff
@@ -789,6 +804,14 @@ groups = []
 
 [customizations]
 installation_device = "/dev/vda"
+
+[[customizations.user]]
+name = "simple"
+description = "Administrator account"
+password = "${EDGE_USER_PASSWORD_SHA512}"
+key = "${SSH_KEY_PUB}"
+home = "/home/simple/"
+groups = ["wheel"]
```

Comment:
`[[customizations.user]]` must be added before `[customizations]` or after adding `fips=true`, otherwise it will break the blueprint definition.

Reply:
Thank you @mmartinv !
I fixed this error and will trigger CI test again.